pkg/controller/job/metrics/metrics.go

```diff
@@ -60,7 +60,7 @@ var (
 	// Possible label values:
 	//   completion_mode: Indexed, NonIndexed
 	//   result:          failed, succeeded
-	//   reason:          "BackoffLimitExceeded", "DeadlineExceeded", "PodFailurePolicy", ""
+	//   reason:          "BackoffLimitExceeded", "DeadlineExceeded", "PodFailurePolicy", "", "Completions", "SuccessPolicy"
```

Comment:
If we drop metrics in Alpha we can revert that

Reply:
Done.